inttest/Makefile.variables

```diff
@@ -78,6 +78,7 @@ smoketests := \
 	check-singlenode \
 	check-stackapplier \
 	check-statussocket \
+	check-windows \
```

Comment:
I'd probably not add this here, as it's a completely different beast as the other smoketests. I'd rather call it directly from the workflow. All it needs is `go test -count=1 -v -timeout 15m github.com/k0sproject/k0s/inttest/windows`.

This would also allow us to avoid the extra special casing in the smoketest workflow.